tests/bluetooth/host/cs/mocks/zephyr/linker/linker-defs.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Undefine definitons that conflict in the unit_testing board build. */
+
+#ifdef DT_NODE_HAS_STATUS
+#undef DT_NODE_HAS_STATUS
+#endif
+
+#ifdef DT_FOREACH_OKAY_HELPER
+#undef DT_FOREACH_OKAY_HELPER
+#endif
+
+#define DT_FOREACH_OKAY_HELPER(x)
```

Comment:
Also wondering about that. Should it be addressed by whoever maintain the unit_testing board?